### PR TITLE
Remove `use_launch_instance_ng` attribute in Horizon

### PR DESCRIPTION
**Pull request by matelakat**
Made the Angular Launch Instance workflow the default in Horizon.

### CODE REVIEW

**Comment by Itxaka** [comment]
+1

http://docs.openstack.org/releasenotes/horizon/mitaka.html

`Made the Angular Launch Instance workflow the default in Horizon.`

File: chef/cookbooks/horizon/templates/default/local_settings.py.erb [hunk 1]
```diff
@@ -249,25 +249,6 @@ OPENSTACK_KEYSTONE_BACKEND = {
...
-# You can choose whether to enable the new launch instance experience,
-# the legacy experience, or both. The legacy experience will be removed
-# in a future release, but is available as a temporary backup setting to ensure
-# compatibility with existing deployments. Further development will not be
-# done on the legacy experience. Please report any problems with the new
-# experience via the Launchpad tracking system.
-#
-# Toggle LAUNCH_INSTANCE_LEGACY_ENABLED and LAUNCH_INSTANCE_NG_ENABLED to
-# determine the experience to enable.  Set them both to true to enable
-# both.
```
Comment (aplanas) [inline chef/cookbooks/horizon/templates/default/local_settings.py.erb]: Does the new default `local_settings.py` from Horizon contains something here?

Comment (matelakat) [inline chef/cookbooks/horizon/templates/default/local_settings.py.erb]: Let me check...

Comment (matelakat) [inline chef/cookbooks/horizon/templates/default/local_settings.py.erb]: https://github.com/openstack/horizon/blob/master/openstack_dashboard/local/local_settings.py.example#L229-L241

**Comment by matelakat** [comment]
[(2) Remove `use_launch_instance_ng` attribute in Horizon](https://trello.com/c/qJKl1bi6/162-2-remove-use-launch-instance-ng-attribute-in-horizon)

**Comment by nkrinner** [comment]
+1

**Comment by vuntz** [comment]
I would probably add in the commit message that we remove it because it's the default in Mitaka, and not experimental anymore.

**Comment by toabctl** [comment]
Would be also nice to prefix the commit msg with ```horizon:``` or  ```dashboard:```

**Comment by vuntz** [comment]
@toabctl we can squash the commits on merge now, so that bit is less of an issue.

**Comment by matelakat** [comment]
rebasing, squashing...

**Pull request merged**